torch/csrc/autograd/profiler_python.cpp

```diff
@@ -77,6 +77,14 @@ PyCodeObject* nnModuleCode() {
   return module_call_code;
 }
 
+template <CallType C>
+PyCodeObject* getCode();
+
+template <>
+PyCodeObject* getCode<CallType::PyModuleCall>() {
```

Comment:
I don't think we need this indirection. We can just move the body and use `getCode<CallType::PyModuleCall>()` everywhere.